src/test/resources/gradle_plugin/pipeline_test_build_scan.txt

```diff
@@ -0,0 +1,12 @@
+stage ('build_Project') {
+    steps {
+        script {
+           if(isUnix()) {
```

Comment:
I needed to replace lines 4-10 with the following to get it running. The path hack below does not work on my machine.

```
        withEnv(["PATH+GRADLE=${tool 'Gradle 4.0'}/bin"]) {
            sh "gradle hello --scan"
        }
```